tests/spread/core24-suites/environment/test-variables/task.yaml

```diff
@@ -33,7 +33,7 @@ execute: |
     cat "$file"
     for exp in \
       "^SNAPCRAFT_PROJECT_GRADE=devel$" \
-      "^SNAPCRAFT_PROJECT_NAME=None$" \
+      "^SNAPCRAFT_PROJECT_NAME=variables$" \
```

Comment:
I think I fixed a bug here by accident

Reply:
Great!  I think this is different than https://github.com/canonical/craft-application/issues/320, which is about `CRAFT_PROJECT_NAME`.